impeller/entity/shaders/glyph_atlas_sdf.frag

```diff
@@ -0,0 +1,36 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+uniform sampler2D glyph_atlas_sampler;
+
+uniform FragInfo {
+  vec2 atlas_size;
+  vec4 text_color;
+  float font_has_color;
+} frag_info;
+
+in vec2 v_unit_vertex;
+in vec2 v_atlas_position;
+in vec2 v_atlas_glyph_size;
+in float v_color_glyph;
```

Comment:
This one isn't used. I think we can't render full color glyphs or bitmaps with SDF anyway so I'd drop it

Reply:
Done

Reply:
I think I'd rather just leave this to avoid replicating a bunch of code on the C++ side for now.